server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/actor/Actor.java

```diff
@@ -0,0 +1,92 @@
+package de.tum.in.www1.hephaestus.codereview.actor;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.hephaestus.codereview.comment.Comment;
+import de.tum.in.www1.hephaestus.codereview.pullrequest.Pullrequest;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.OneToMany;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "actors")
```

Comment:
In my opinion both are fine, it's just a matter of convention. 
I've mostly used the plural in the past because makes the separation between the table and attributes that reference the table more clear.
I renamed it to the singular now to stay consistent with Artemis (bae8980).